Files/UserControls/Widgets/Bundles.xaml

```diff
@@ -0,0 +1,529 @@
+                                        </MenuFlyout>
+                                    </Button.Flyout>
+                                </Button>
+                            </Grid>
+
+                            <!--  List of Bundle Items  -->
+                            <GridView
+                                HorizontalAlignment="Stretch"
+                                VerticalAlignment="Stretch"
+                                ItemsSource="{Binding Contents, Mode=OneWay}"
+                                SelectionMode="None"
+                                Grid.Row="1">
+                                <GridView.ItemTemplate>
+                                    <!--  Bundle Item design  -->
+                                    <DataTemplate>
+                                        <Grid>
+                                            <Button
+                                                Width="119"
+                                                Height="36"
+                                                Margin="2,0,2,0"
+                                                HorizontalAlignment="Stretch"
+                                                VerticalAlignment="Stretch"
+                                                HorizontalContentAlignment="Stretch"
+                                                VerticalContentAlignment="Center"
+                                                Background="Transparent"
+                                                BorderThickness="1"
+                                                Command="{Binding OpenItemCommand, Mode=OneWay}">
+
+                                                <Grid ColumnSpacing="4">
+                                                    <Grid.ColumnDefinitions>
+                                                        <ColumnDefinition Width="Auto" />
+                                                        <ColumnDefinition />
+                                                    </Grid.ColumnDefinitions>
+
+                                                    <!--  Icon  -->
+
+                                                    <!--  File Icon  -->
+                                                    <Image
+                                                        Grid.Column="0"
+                                                        Width="20"
+                                                        Height="20"
+                                                        HorizontalAlignment="Left"
+                                                        VerticalAlignment="Center"
+                                                        Source="{Binding Icon, Mode=OneWay}"
+                                                        Visibility="{Binding FileIconVisibility}" />
+                                                    <!--  Folder Icon  -->
+                                                    <Image
```

Comment:
This second `Image` could be removed. Change type of `Icon` VM property to `ImageSource`. Remove `FileIconVisibility` flag too. Add property `FolderIcon` and use it in `SetIcon()`.
```csharp
public SvgImageSource FolderIcon { get; } = new SvgImageSource()
{
    RasterizePixelHeight = 128,
    RasterizePixelWidth = 128,
    UriSource = new Uri("ms-appx:///Assets/FolderIcon.svg"),
};
```

Reply:
@d2dyno1 Will you implement it?

Reply:
I didn't quite understand the way you presented it. Could you provide more details? @tsvietOK